### ISSUE TITLE
can't make clearml-init to work when using chart

### ISSUE THREAD

**Issue by awaizman1** [allegroai/clearml#910]
Hi,

I'm creating a kind cluster, port-forwarding clearml ports using extraPortMappings (as described in documentation):
```
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
  - role: control-plane
    extraPortMappings:
      # clearml api-server
      - containerPort: 30008
        hostPort: 30008
      # clearml web-server
      - containerPort: 30080
        hostPort: 30080
      # clearml file-server
      - containerPort: 30081
        hostPort: 30081
```

using the helm chart i'm installing the clearml server and agent successfully, also being able to open the web application via ```http://localhost:30080``` as expected.

Now i'm trying to setup my host to connect to clearml using ```clearml-init```. once prompt to enter credentials, i copy it from server but it seems there is some parsing bug and the reported api-server url is wrong (note that although the provided configuration mention that ```api_server: http://localhost:30008```, the final parsed configuration is wrong ```API: http://localhost:30080``` - wrong port).

```bash
(venv) ➜  clearml-test clearml-init                                                                                               ⎈|kind-kind:default
ClearML SDK setup process

Please create new clearml credentials through the settings page in your `clearml-server` web app (e.g. http://localhost:8080//settings/workspace-configuration) 
Or create a free account at https://app.clear.ml/settings/workspace-configuration

In settings page, press "Create new credentials", then press "Copy to clipboard".

Paste copied configuration here:
api { 
    web_server: http://localhost:30080
    api_server: http://localhost:30008
    credentials {
        "access_key" = "FPP3NSWYA36G1RH32WCB"
        "secret_key"  = "PmlLJ3ya7otojDYyxNxNdMuRceqvRddTD8zZo2vbdK1adrJ8or"
    }
}
Detected credentials key="FPP3NSWYA36G1RH32WCB" secret="PmlL***"

ClearML Hosts configuration:
Web App: http://localhost:30080
API: http://localhost:30080
File Store: 

Verifying credentials ...
Error: could not verify credentials: key=FPP3NSWYA36G1RH32WCB secret=PmlLJ3ya7otojDYyxNxNdMuRceqvRddTD8zZo2vbdK1adrJ8or
```

Following above configuration, i can't proceed with passing the clearml-init command.
I validated the code in clearml.cli.config and indeed seems there is parsing logic bug there.

Thanks.

**Comment by awaizman1** [comment]
hi @valeriano-manassero,

where can i find the transferred bug?

**Comment by valeriano-manassero** [comment]
Still this bug but now it's on clearml repo 😄

**Comment by khongtrunght** [comment]
Any update yet ?

**Comment by ainoam** [comment]
@khongtrunght A fix should be coming soon. In the mean time you should be able to circumvent this issue by manually creating your own clearml.conf (as was suggested [here](https://github.com/allegroai/clearml-server/issues/181))

**Comment by pollfly** [comment]
Hey @awaizman1 and @khongtrunght ! Just letting you know that this issue has been resolved in the recently released [v1.10.0](https://github.com/allegroai/clearml/releases/tag/v1.10.0). Let us know if there are any issues :)

**Comment by jkhenning** [comment]
@awaizman1, @khongtrunght, closing this, please reopen if required.
